Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanelModifier.swift

```diff
@@ -51,7 +51,7 @@ public extension View {
                 horizontalAlignment: horizontalAlignment,
                 isPresented: isPresented,
                 maxWidth: maxWidth,
-                panelContent: content()
```

Comment:
This is the other main piece of the change. The content was being evaluated right here in the modifier with no regards to the value of `isPresented`.